lms/views/api/canvas/files.py

```diff
@@ -1,10 +1,18 @@
 """Proxy API views for files-related Canvas API endpoints."""
+import re
+
 from pyramid.view import view_config, view_defaults
 
 from lms.security import Permissions
 from lms.services.canvas import CanvasService
 from lms.views import helpers
 
+#: A regex for parsing just the file_id part out of one of our custom
```

Comment:
```suggestion
#: A regex for parsing just the FILE_ID part out of one of our custom
```

Reply:
Reworded, we get both FILE_ID and COURSE_ID.